src/Provider/ActiveMQ.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+namespace Riverline\WorkerBundle\Provider;
+
+use Riverline\WorkerBundle\Queue\Queue;
+use Stomp\Client;
+use Stomp\Network\Connection;
+use Stomp\StatefulStomp;
+use Stomp\Transport\Message;
+
+/**
+ * Class ActiveMQ
+ * @package Riverline\WorkerBundle\Provider
+ */
+class ActiveMQ extends AbstractBaseProvider
+{
+    /**
+     * @var StatefulStomp
+     */
+    private $stomp;
+
+    /**
+     * @var Client
+     */
+    private $client;
+
+    /**
+     * ActiveMQ constructor.
+     *
+     * @param string $brokerUri
+     * @param string $login
+     * @param string $password
+     */
+    public function __construct($brokerUri, $login, $password)
+    {
+        $connection = new Connection($brokerUri);
+        $connection->setReadTimeout(5, 0);
+
+        $this->client = new Client($connection);
+        $this->client->setLogin($login, $password);
+        $this->client->setSync(true);
+
+        $this->stomp = new StatefulStomp($this->client);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function count($queueName)
+    {
+        $tempQueueName = sprintf("/temp-queue/%s", uniqid());
+
+        $this->client->sendFrame(new Message(
```

Comment:
Do you have to create the queue ? The plugin don't create it ?

Reply:
Exactly